bigbluebutton-html5/imports/api/breakouts/server/handlers/breakoutStarted.js

```diff
@@ -0,0 +1,38 @@
+import Breakouts from '/imports/api/breakouts';
+import Logger from '/imports/startup/server/logger';
+import { check } from 'meteor/check';
+
+export default function ({ payload }) {
+  const meetingId = payload.meetingId;
+
+  check(meetingId, String);
+
+  const selector = {
+    breakoutMeetingId: payload.meetingId,
+  };
+
+  modifier = {
+    $set: {
+      users: [],
+      timeRemaining: Number(payload.timeRemaining),
```

Comment:
Please add a constant for payload.timeRemaining, payload.externalMeetingId and check them. Also please use meetingId instead of payload.meetingId